### PR TITLE
Add users recommendations to the profile page

### PR DESCRIPTION
**Pull request by BryanTriana**
The profile page with the users recommendations looks like this:

UserRecommendationsProfilePage

If a user shows up in the user recommendation list it means that the user has liked some of the same items as you. The order of the recommended users also has a priority (The first ones have more shared likes than the last ones on the list)

### CODE REVIEW

**Comment by BryanTriana** [comment]
Right now there are no links to the recommended users since that change would need to modify the way user_script loads the profile page. I am planning to add this change on a future PR, once I discuss it with Oyin.

**Comment by BryanTriana** [comment]
OK so I am guessing I should add like another div with some small text that explains the rankings.

File: src/main/webapp/ProfilePage.html [hunk 1]
```diff
@@ -44,15 +45,17 @@ <h6 class="card-subtitle mb-2" id="username"></h6>
...
+        <div class="mt-3">
           <h2 class="text-center">Favorite Items</h2>
           <div
-            class="card"
             style="width: 45em; margin: 0 auto;"
             style="height: 10rem;"
           >
             <div id="item-container"></div>
           </div>
+          <h2 class="text-center mt-3">Recommended Users</h2>
```
Comment (codewithjess) [inline src/main/webapp/ProfilePage.html:56]: should we hide this text if there are no recommended users?

File: src/main/webapp/user_script.js [hunk 2]
```diff
@@ -114,3 +111,50 @@ function createFavoriteItemCard(entertainmentItem) {
...
+ * Creates a card div that displays the user avatar and the username.
+ *
+ * @param { JSON } profile - the JSON object holding data about a user profile
+ * @returns { jQuery } card element representing the user
+ */
+function createProfileCard(profile) {
+  const card = $('<div class="card bg-light"></div>');
+  card.append(
+      $('<img class="card-img-top" src="https://icotar.com/avatar/' +
+        profile.username.charAt(0) + '">'));
```
Comment (codewithjess) [inline src/main/webapp/user_script.js:151]: Lets move the avatar url logic  `"https://icotar.com/avatar/" + profile.username.charAt(0)`  into a helper function and call that function in loadProfile() line 31

This guarantees that all parts of the codebase use the same logic to create the URLs.

File: src/main/webapp/ProfilePage.html (hunk 1 above)
Comment (BryanTriana) [reply src/main/webapp/ProfilePage.html:56]: I decided to keep the text, but I added dynamic text that tells the user to like more items if this section is empty.

File: src/main/webapp/user_script.js (hunk 2 above)
Comment (BryanTriana) [reply src/main/webapp/user_script.js]: OK I separated the avatar url logic into a separate function in last commit.

**Comment by BryanTriana** [comment]
Here is how the page looks now with the new changes:

Screenshot 2020-07-23 at 3 44 09 PM

And this is how it looks when there are no recommended users:

Screenshot 2020-07-23 at 3 43 12 PM

File: src/main/webapp/ProfilePage.html (hunk 1 above)
Comment (codewithjess) [reply src/main/webapp/ProfilePage.html:56]: nice I like that idea :)

**Pull request merged**